src/schema/v2/artworksForUser/artworksForUser.ts

```diff
@@ -28,6 +29,7 @@ export const artworksForUser: GraphQLFieldConfig<void, ResolverContext> = {
     userId: { type: GraphQLString },
     version: { type: GraphQLString },
     maxWorksPerArtist: { type: GraphQLInt },
+    marketable: { type: GraphQLBoolean, defaultValue: false },
```

Comment:
**suggestion**: Thoughts on making this nullable or unspecified? It feels like with this default of `false`, any queries not specifying an explicit `marketable` value would not receive any marketable works. Is that the behavior we want for currently executing queries?